guides/v2.2/magento-functional-testing-framework/release-2/test/annotations.md

```diff
@@ -195,6 +198,28 @@ Attribute|Type|Use
 <useCaseId value="USECASE-1"/>
 ```
 
+### skip
+
+The `<skip>` element is used to skip a test.
```

Comment:
Use the `<skip>` element to skip a test.

Reply:
🆗

Reply:
🆗